### PR TITLE
Do not attempt scheduling if Kube integrator is not ready or healthy

### PR DESCRIPTION
**Pull request by tbak**

### CODE REVIEW

File: titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/batch/BatchDifferenceResolver.java [hunk 1]
```diff
@@ -269,17 +269,19 @@ public BatchDifferenceResolver(
...
-                    missingTasks.add(BasicTaskActions.launchTaskInKube(
-                            kubeApiServerIntegrator,
-                            configuration,
-                            engine,
-                            runningJobView.getJob(),
-                            refTask,
-                            RECONCILER_CALLMETADATA.toBuilder().withCallReason("Launching task in Kube").build(),
-                            titusRuntime
-                    ));
+                if (kubeApiServerIntegrator.isReadyForScheduling()) {
```
Comment (fabiokung) [inline titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/batch/BatchDifferenceResolver.java:272]: minor: combine the `if` below into a single boolean logic operation (`&&`)

Comment (fabiokung) [reply titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/batch/BatchDifferenceResolver.java:272]: even better, maybe consider extracting the boolean logic chain into a separate method/function

File: titus-server-master/src/main/java/com/netflix/titus/master/scheduler/TaskPlacementFailureClassifier.java [hunk 2]
```diff
@@ -153,6 +155,23 @@ private boolean processNoActiveAgent(T taskRequest, List<TaskAssignmentResult> a
         return true;
     }
 
+    private boolean processKubeAbiNotReady(T taskRequest, List<TaskAssignmentResult> assignmentResults,
```
Comment (fabiokung) [inline titus-server-master/src/main/java/com/netflix/titus/master/scheduler/TaskPlacementFailureClassifier.java:158]: typo `s/Abi/Api/`

File: titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/batch/BatchDifferenceResolver.java (hunk 1 above)
Comment (tbak) [reply titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/batch/BatchDifferenceResolver.java:272]: I was thinking about extracting this into a util function, but there would be too many arguments, which would not improved the code readability.

**Pull request merged**